challenges/objects-arrays.js

```diff
@@ -84,42 +115,45 @@ zooAnimals = [{"animal_name":"Jackal, asiatic","population":5,"scientific_name":
 {"animal_name":"Australian pelican","population":5,"scientific_name":"Pelecanus conspicillatus","state":"West Virginia"}];
 
 /* Request 1: .forEach()
-
 The zoo wants to display both the scientific name and the animal name in front of the habitats.  Return an array with only the animal and scientific names in it.  The individual values in the array should look like this "Name: Jackal, asiatic, Scientific: Canis aureus."
-
 */
 const animalNames = [];
+zooAnimals.forEach((animal) => {
+
+ animalNames.push(`Name: ${animal.animal_name}, Scientific: ${animal.scientific_name}`)
+});
 console.log(animalNames);
 
 /* Request 2: .map()    
-
 The zoos need a list of all their animal's names (names only, not scientific) converted to lower case.  Create a new array named lowerCase and map over each name to convert them all to lower case.  Log the resut.
-
 */
 
-const lowerCase = [];
+const lowerCase = zooAnimals.map(names =>
+  names.animal_name.toLowerCase()
+);
 console.log(lowerCase); 
 
 /* Request 3: .filter() 
-
 The zoos are concenred about animals with a lower population count. Find out which animals have a population less than 5.
-
 */
-const lowerPopulation = [];
+const lowerPopulation = zooAnimals.filter((animal) => {
+  return animal.population <= 5;
+});
```

Comment:
make sure to read the instructions carefully, the challenge here is asking for populations less than 5, see if you can catch the mistake.